### ISSUE TITLE
Mobs that die to fire damage trigger ragdolls late [BUG]

### ISSUE THREAD

**Issue by BLUEChester** [haubna/PhysicsMod#524]
**Describe the bug**
When a mob dies to fire damage it dies in the normal vanilla way and  then triggers the ragdoll effect

**To Reproduce**
Simply let a zombie out in the daylight and wait for it to die

**Screenshots**

https://user-images.githubusercontent.com/68225807/212246825-a830b737-69a1-4a79-b430-20873fcb9328.mp4

Minecraft Version: Quilt 1.19.2
Physics Mod Version: V103

**Comment by haubna** [comment]
Might be due to another mod interfering, I can't remember that this happened in my vanilla tests.

**Comment by BLUEChester** [comment]
Yeah definitely possible, im unable to get a mod list for a few day but once im free ill send a mod list here
